packages/app/src/features/mainPage/components/Folder.tsx

```
@@ -0,0 +1,122 @@
+        )}
+      </div>
+      <div className={`flex items-center ${props.isRoot ? "ml-7.5" : "ml-13.5"}`}>
+        {props.mode === "display" && (
+          props.folder.name
+        )}
+        {props.mode === "input" && (
+          <input
+            type="text"
+            defaultValue={""}
+            autoFocus
+            onClick={(e) => e.stopPropagation()}
```

Biome (2.1.2)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)